proof_ipa.go

```diff
@@ -226,10 +203,9 @@ func MakeVerkleMultiProof(preroot, postroot VerkleNode, keys [][]byte, resolver
 	return proof, pe.Cis, pe.Zis, pe.Yis, nil
 }
 
-// VerifyVerkleProofWithPreAndPostTrie takes a pre-state trie, a post-state trie and a list of keys, and verifies that
-// the provided proof verifies. If the post-state trie is `nil`, the behavior is the same as `VerifyVerkleProof`.
-func VerifyVerkleProofWithPreAndPostTrie(proof *Proof, preroot, postroot VerkleNode) error {
-	_, _, _, _, proof_cis, _, proof_ys, proof_zs, err := getProofElementsFromTree(preroot, postroot, proof.Keys, nil)
+// VerifyVerkleProofWithPreState takes a proof and a trusted tree root and verifies that the proof is valid.
+func VerifyVerkleProofWithPreState(proof *Proof, preroot VerkleNode) error {
+	_, _, _, _, proof_cis, _, proof_ys, proof_zs, err := getProofElementsFromTree(preroot, nil, proof.Keys, nil)
```

Comment:
so yeah, get `pe` here and use it line 213. My suggested change above won't compile by itself.